api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/summary/FormatUtils.kt

```diff
@@ -0,0 +1,261 @@
+package org.jetbrains.kotlinx.dl.api.summary
+
+import kotlin.math.max
+
+/**
+ * Convenient function to calculate the required width of the column to fit all the values.
+ * @param [headerWidth] width of the column header
+ * @param [rows] list of rows to calculate the width
+ */
+public fun calcColumnWidth(headerWidth: Int, rows: List<String>): Int {
+    return rows.maxOfOrNull { it.length }?.coerceAtLeast(headerWidth) ?: headerWidth
+}
+
+/**
+ * Convenient function to calculate the required width of all columns.
+ * @param [rows] table with multiple rows each with multiple columns
+ * @param [columnNames] list of column names
+ *
+ * Note: each row must have the same number of columns as the [columnNames] list.
+ * Note: each row may consist of multiple lines.
+ */
+public fun calcColumnWidths(rows: List<TableRow>, columnNames: List<String> = emptyList()): MutableList<Int> {
+    if (columnNames.isEmpty()) {
+        return listOf(rows.flatMap { it.lines() }.maxOfOrNull { it.size } ?: 0).toMutableList()
```

Comment:
`calcColumnWidths(listOf(TableRow(listOf("one", "two", "three"))))` returns `[3]` for me. Is this correct?